docs/src/prototypes/chatPane/chatPaneHeader.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Layout, Avatar, Text, Button, Segment, Icon, Divider } from '@stardust-ui/react'
+
+import { IChat } from './data/interfaces'
+
+export interface IChatPaneHeaderProps {
+  chat?: IChat
+}
+
+class ChatPaneHeader extends React.PureComponent<IChatPaneHeaderProps> {
+  public render() {
+    return (
+      <Layout
+        vertical
+        start={this.renderBanner()}
+        main={this.renderMainArea()}
+        end={
+          <Divider
+            type="secondary"
+            styles={{ root: { marginTop: 0, marginBottom: 0, width: '100%' } }}
+          />
+        }
+      />
+    )
+  }
+
+  private renderBanner(): React.ReactNode {
+    return (
+      <Segment
+        content={
+          <Icon
+            svg
+            name="umbrella"
+            styles={{ root: { margin: 'auto 0' } }}
+            variables={siteVars => ({ color: siteVars.white })}
+          />
+        }
+        styles={{
+          root: ({ variables: v }) => ({
+            backgroundColor: v.backgroundColor,
+            borderRadius: 0,
+            display: 'flex',
+            height: '40px',
+            width: '100%',
+          }),
+        }}
+        variables={siteVars => ({ backgroundColor: siteVars.brand })}
+      />
+    )
+  }
+
+  private renderMainArea(): React.ReactNode {
+    const { chat } = this.props
+
+    return (
+      <Layout
+        start={<Avatar name={chat.title} />}
+        main={
+          <Text
+            size="lg"
+            content={chat.title}
+            styles={{ root: { marginLeft: '12px', fontWeight: 600 } }}
+          />
+        }
+        end={this.renderHeaderButtons()}
+        alignItems="center"
+        styles={{ root: { padding: '16px 32px' }, end: { display: 'inline-flex' } }}
+      />
+    )
+  }
+
+  private renderHeaderButtons(): React.ReactNode {
```

Comment:
since we have it already, it would be nice to add the example of popup here

Reply:
I'd rather not do it in this PR before we make a decision about the current `Popup` issues #263